### DISCUSSION TITLE
Screen going dark

### DISCUSSION THREAD

**Discussion by Editpo** [rern/rAudio#1730] [Bug Reports]
Since updating to 20240719 when I het any of the menu items top right on playlist page the screen goes dark. You can just about see what’s there but can’t get any further. Opening up a new tab seems to bring it back but it happens everytime. Using iPad

**Comment by Wheatleybloke** [comment]
Same issue phone and desktop; anychanges to playlist means a refresh of screen

**Reply by Editpo** [reply]
Strange. mine works ok on phone

**Comment by Editpo** [comment]
Getting this also on 20240818.
I believe it is because when you hit a button that brings up a dialogue the dialogue is off screen somewhere leaving the screen dimmed. Same on phone, iPad  and laptop.
Edit: if there are only a couple of songs in the playlist it displays ok and when you hit the delete button you get the dialogue. As soon as the number of songs reaches a screen full then it goes wrong

**Reply by klassikmann** [reply]
@Editpo 
Your observation and interpretation of the issue is correct. You have a chance to see the delete dialogue deeply below if you heavely reduce the zoom factor in the browser.
Edit: cf. my posting https://github.com/rern/rAudio/discussions/1720
